### ISSUE TITLE
TensorBoard: display plot as a function of nb epoch and as nb of steps

### ISSUE THREAD

**Issue by tarrade** [tarrade/proj_multilingual_text_classification#8]
display plot as a function of nb epoch (this is wrong, we have 2 epochs here!)
![image](https://user-images.githubusercontent.com/12021701/77640221-4dba0d80-6f5a-11ea-948e-ff5dea826474.png)

this display only the first value (acc, loss) -> wrong
Epoch 1/10
2/2 [==============================] - 70s 35s/step - loss: **0.6915** - accuracy: **0.5469**

and as nb of steps (as for tf.estimator)

**Comment by tarrade** [comment]
just need to use the right parameters in TensorBoard:

tf.keras.callbacks.TensorBoard(log_dir=log_dir, 
                                                      histogram_freq=1, 
                                                      embeddings_freq=1,
                                                      write_graph=True,
                                                      update_freq='batch')
